### DISCUSSION TITLE
Add shortcut key for  "tap and drag"

### DISCUSSION THREAD

**Discussion by AjianNie** [noah-nuebling/mac-mouse-fix#826] [Other]
I like to set "tap and drag" to "Mission Control" and "Spaces," but in actual usage, this action causes lag when swiping between desktops – because I'm used to dragging for longer periods, whereas the default requires a shorter tap and drag.

So, it would be better either optimizing the long tap and drag or simply supporting a shortcut key for this function. And the system default shortcut for Mission Control and Spaces is control+left/right arrow keys.

**Comment by EusebiuSeby** [comment]
100%. I noticed that too. Click and Drag gesture should support shortcuts as well. It is a very powerful gesture but is sad that is limited to scroll only actions.
